docs/app.config.ts

```diff
@@ -16,6 +16,7 @@
 
 export default defineAppConfig({
   exactproDocs: {
-    title: 'Docs Template Project'
+    title: 'Docs Template Project',
+    hostname: 'https://example.com'
```

Comment:
Hostname should be defined in root app.config. Here you should specify concrete path for docs: `https://exactpro.github.io/docs-toolkit`

And I think it is better to specify option by following path: `exactproDocs.seo.sitemap.baseUrl`